src/html.rs

```diff
@@ -0,0 +1,111 @@
+use std::collections::{BTreeMap, BTreeSet};
+
+use maplit::{btreemap as map, btreeset as set};
+use once_cell::sync::Lazy;
+use scraper::{ElementRef, Html, Selector};
+
+static BAD_SECTIONS: Lazy<BTreeMap<&str, BTreeSet<&str>>> = Lazy::new(|| {
+    map! {
+        "de" => set!{
+            "Einzelnachweise",
+            "Weblinks",
+            "Literatur",
+            "Siehe auch",
+            "Anmerkungen",
+            "Anmerkungen und Einzelnachweise",
+            "Filme",
+            "Einzelbelege",
+        },
+        "en" => set!{
+            "External links",
+            "Sources",
+            "See also",
+            "Bibliography",
+            "Further reading",
+            "References",
+        },
+       "es" => set!{
+            "Vínculos de interés",
+            "Véase también",
+            "Enlaces externos",
+            "Referencias"
+        },
+        "fr" => set!{
+            "Bibliographie",
```

Comment:
Is there a more robust way to exclude some sections for all languages?

Reply:
No need to collapse, a config looks like a better option (keep adding many other languages in mind, and potential contributors who doesn't read rust code)

Reply:
I added a compile-time json config in 7e6b39a, adding in a flag for loading one at runtime is straightforward if we want to do that later. Using a different config language is also a quick change.